libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts

```diff
@@ -175,6 +188,28 @@ export class TigerWorkspaceDashboards implements IWorkspaceDashboardsService {
         });
     };
 
+    private getFilterContextFromExportId = async (exportId: string): Promise<IFilterContext> => {
+        const result = await this.authCall((client) => {
+            return client.export.getMetadata({
+                workspaceId: this.workspace,
+                exportId,
+            });
+        }).then((result) => result.data);
+
+        // metadata is under our control, accepts arbitrary json, currently we
+        // store there only `filters` array for the export-based filters
+        const { filters } = result as any as { filters: FilterContextItem[] };
```

Comment:
Maybe we could create a convertor here so that these casts are in one place for future use.

Reply:
Added to converters, it's "larger" change so please check d87a4e2e0270a585b8ea7215082d44ddb827824c

Reply:
Thank you for this :)